webapp/components/general/NetPyNEPythonConsole.js

```diff
@@ -104,10 +104,25 @@ export class NetPyNEPythonConsole extends Component {
 
   render () {
     const notebookName = GEPPETTO_CONFIGURATION.notebookName || "notebook.ipynb";
+
+      // Define the style to hide the iframe by positioning it off-screen
+      const iframeStyle = {
+        position: 'absolute',
+        left: '-10000px',
```

Comment:
I guess the goal is move this outside the visible part of the screen, in that case I would use window.innerWidth to get the screensize and move this outside that. Thx